CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 The format of this file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Radar: Updated User Interface of WebGUI
```

Comment:
Like the CodeCommit bot says, there should be an empty line here between L10 and L11.